### DISCUSSION TITLE
category lists are not showing in version 9 after migration from version 8

### DISCUSSION THREAD

**Discussion by jeffalgarne** [jhuckaby/Cronicle#616] [Q&A] [answered]
We are on the process of migrating to version 9 but after copying the data from version 8 the category lists are not showing and it only shows the first category the other categories are missing.

**Comment by jeffalgarne** [comment]
Version 8
![image](https://github.com/jhuckaby/Cronicle/assets/4668227/b0533ba5-dee1-4733-8147-373fc3a49dba)

Version 9
![image](https://github.com/jhuckaby/Cronicle/assets/4668227/6d72f513-cfdd-4265-94a9-5cb5711b496d)

**Comment by jhuckaby** [comment, marked as answer]
It looks like some data corruption occurred during your data transfer.  Make sure you follow the [guide here](https://github.com/jhuckaby/Cronicle/blob/master/docs/CommandLine.md#data-import-and-export), and make sure that Cronicle is fully stopped and not running when you export and import the data.

**Reply by jeffalgarne** [reply]
@jhuckaby Thanks the guide works
